runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java

```diff
@@ -55,12 +55,22 @@ default boolean highlightDestinationTile()
 	{
 		return true;
 	}
+	@ConfigItem(
+		keyName = "destinationTileFillColor",
+		name = "Destination tile fill color",
+		description = "Configures the fill color of destination tile",
+		position = 3
+	)
+	default boolean destinationTileFillColor()
```

Comment:
all of these should probably have `@Alpha`

Reply:
Sounds good ill update and do another commit with that